virtuaaliviivakoodi/deconstructors/amount.py

```diff
@@ -0,0 +1,11 @@
+import decimal
+
+
+def deconstruct_amount(virtuaaliviivakoodi: str) -> decimal.Decimal:
```

Comment:
Now, as the deconstructed amount is Decimal, I'd like to see the original virtuaaliviivakoodi-function to accept decimals. Just because of symmetry 😄

Reply:
Added the support for this. Had to change the comparison in `validators/euro_amount.py` to use decimals instead of float. Previously the comparison was against float of `99999.99` which actually is around`999999.98999999999068677425384521484375`. This would not work when comparing to the max amount as the float is not quite `999999.99`